src/docs/Ways of Working/percy.md

```diff
@@ -0,0 +1,166 @@
+---
+sidebar_position: 5
+slug: /ways-of-working/percy
+---
+
+# Percy Visual Testing
+
+## Intruduction 
+
+This document describes how you can setup a Percy project, connect it with a GitHub repository and how to use it from the command line or as part of the review builds.
+
+## Prerequisites
+
+- Access to [gov-cy](https://github.com/gov-cy) organization settings on Github 
+- [Percy Github App](https://github.com/apps/percy) installed for the `gov-cy` organization 
+- Access to [Percy](https://percy.io/)
+- nmp package [@percy/script](https://docs.percy.io/docs/percyscript) is installed
+
+## Steps to set it up
+
+1. Give permissions on Percy Github App to access the desired repository
+	- Go to https://github.com/organizations/gov-cy/settings/installations and click `Configure` for the Percy App
+	- Under the `Repository Access` Click on `Select repositories` and select the desired repository.
+	- Click `Save`
+	
+	![Github permissions](../assets/percy1.png)
+ 
+2.	Create new project in Percy
+	- Go to Percy's organization page (https://percy.io/a0d81626)
+	- Click `Create new project`
+	- Complete the `Project name` and `Link a repository` (see image below)
+	- Click `Create Project`
+	
+    ![Percy create new project](../assets/percy2.png)
+	
+3. Set PERCY_TOKEN in repository secret
+	- From the Percy project page copy the `PERCY_TOKEN` value.
+	
+	![PERCY_TOKEN on percy page](../assets/percy3.png)
+	
+	- Go to repository's settings page in Github, click on `Secrets` (i.e. https://github.com/gov-cy/dsf-design-system/settings/secrets/actions URL is different depending on repository)
+	- Click `New repository secret` and enter under *Name* `PERCY_TOKEN` and under *Value* paste the token value.
+	
+	![Repository secret](../assets/percy4.png)
+
+4. Create npm test script 
+    - In this step you will need to create a test script that will be used to generate the screenshots and add them to Percy. Depending on the technology used (such as Ember, Cypress, Capybara, Storybook e.t.c) Percy has *how to* tutorials at https://docs.percy.io/docs/example-apps. In this tutorial we are using [PercyScript](https://docs.percy.io/docs/percyscript) which uses the `@percy/script` npm package.
+    - Create a test script under `test\scripts\percy.js`. See below a sample test script
+
+```js 
+const PercyScript = require('@percy/script');
+const httpServer = require('http-server');
+
+const PORT = process.env.PORT_NUMBER || 3000;
+const TEST_URL = `http://localhost:${PORT}/build`;
+
+// A script to navigate our app and take snapshots with Percy.
+PercyScript.run(async (page, percySnapshot) => {
+  let server = httpServer.createServer();
+  server.listen(PORT);
+
+  console.log(`Server started at ${TEST_URL}`);
+
+  await page.goto(TEST_URL);
+  await percySnapshot('index',{ widths: [375, 767, 1280] });
+
+  server.close();
+});
+
```

Comment:
An example of how to add another page here or a link to the Percy docs on how to do this would be useful

Reply:
It's something like:

```
  await page.goto(TEST_URL + '/my-cool-new-page');
  await percySnapshot('Cool New Page',{ widths: [375, 767, 1280] });
```

Reply:
added it in the sample code and added some explanation (line 50)

Reply:
👍